executorlib/interactive/shared.py

```
@@ -260,44 +121,108 @@ def shutdown(self, wait: bool = True, *, cancel_futures: bool = False):
-        try:
-            item = que.get_nowait()
-            if isinstance(item, dict) and "future" in item.keys():
-                item["future"].cancel()
-                que.task_done()
-        except queue.Empty:
-            break
+
+    def __init__(
+        self,
+        max_workers: int = 1,
+        executor_kwargs: dict = {},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)